### PR TITLE
Fix #1082: AudioNode context is a BaseAudioContext

### LINKED ISSUE #1082: AudioNode "context" attribute is type of BaseAudioContext
But, the below document is not edited from AudioContext yet.

https://webaudio.github.io/web-audio-api/#the-audionode-interface

```ruby
interface AudioNode : EventTarget {
  // ..
  readonly        attribute AudioContext          context;
  // ..                      ^
}                            |
                            BaseAudioContext
```

```
2.4.1 Attributes

context of type AudioContext, readonly
    The AudioContext which owns this AudioNode.
         ^       ^
         |       |
        BaseAudioContext
```

### PR DESCRIPTION
**Pull request by rtoy**
Fix typo.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2824,11 +2824,11 @@ <h2 id="OfflineAudioCompletionEvent">
...
           </dd>
           <dt>
-            readonly attribute AudioContext context
+            readonly attribute BaseAudioContext context
           </dt>
           <dd>
             <p>
-              The <a><code>AudioContext</code></a> which owns this
+              The <a><code>BaseAudioContext</code></a> which owns this
               <a><code>AudioNode</code></a>.
```
Comment (hoch) [inline index.html]: I don't think we need `<code>` tags here.

Comment (rtoy) [inline index.html]: I think we do this almost everywhere.  If this is the preferred style, we need to examine the entire spec to make it consistent.  So, I'd prefer to do that separately.

Comment (hoch) [inline index.html]: SGTM.

**Comment by joeberkovitz** [comment]
Looks fine - I say merge!

**Pull request merged**